repo-config/package.json

```diff
@@ -45,11 +42,10 @@
     "mkdirp": "^0.5.1",
     "prettier": "^2.4.1",
     "react": "17.0.2",
-    "react-native-codegen": "^0.0.13",
+    "react-native-codegen": "^0.0.17",
     "react-test-renderer": "17.0.2",
-    "shelljs": "^0.8.5",
+    "shelljs": "^0.8.4",
     "signedsource": "^1.0.0",
-    "typescript": "4.4.2",
```

Comment:
It seems we also had typescript here and on lines 17-18. Those might be worth keeping, but the `react-native-macos-init` package also pulls it in so maybe we don't need to.